test/validate/eia_test.py

```diff
@@ -46,17 +46,17 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_642_829, 1_642_829, 135_980),
+        ("bf_eia923", 1_642_806, 1_642_806, 135_980),
         ("bga_eia860", 153_487, 153_487, 153_487),
-        ("boil_eia860", 89_051, 89_051, 89_051),
+        ("boil_eia860", 89_050, 89_050, 89_050),
```

Comment:
do you have a sense of why these went down? the bf in particular here seems wrong because there should be a few more months of data in there for the raw and monthly tables

Reply:
The new months of 923 data show up as data inputs into existing columns. Once we hit January there will be rows (transformed from the raw table columns) for every month, regardless of whether there is data in them. Row fluctuations are more likely due to retroactive data changes which can lead to positive and negative changes.

I dug a little deeper in to the bf table and uncovered some unexpected cruft that explains some of the row count changes. See my comment below!